### DISCUSSION TITLE
disable or toggle-enable 'confirm delete'

### DISCUSSION THREAD

**Discussion by aum7** [Audiveris/audiveris#766] [Ideas]
as a suggestion : 

the dialog to confirm deletion is not needed
we do have undo
beside, what are the chances to accidentally hit 'delete' button on your keyboard ? close to 0

thx for sharing 

have fun

**Comment by hbitteur** [comment]
The prompt is used only when there are **several** inters to delete.
The user is not prompted for a **single** delete.
So it is not really the question of hitting the delete button, but the question of the _multiplicity_ of the impacted inters.

This being said, we could add the ability for the user to keep the prompt or not. A choice that could be reverted at any time.
What do you think?

**Comment by aum7** [comment]
well, if it is not much trouble
i would still like to be able to chose / toggle
confirmation dialog on / off

**Comment by mls111888** [comment]
I realize this is an old post, but this is something I would be interested in. My first task after transcription is to delete all of the text. I work mostly with choral pieces and there are usually many unwanted elements in the lyrics (staccato, tenuto, etc.) I add all the text back in when I import it into MuseScore. It would help with flow if there wasn't an extra key press. As mentioned previously, there is always undo.

**Comment by hbitteur** [comment]
@mls111888 
OK, I'll take care of it

**Comment by hbitteur** [comment]
Starting with 5.10.1, you can now choose to avoid any prompt about "multiple delete"
See the Preferences dialog.

**Comment by hbitteur** [comment]
I think we can close this discussion, since 5.10.1 is out
